tests/test_hierarchy.cc

```diff
@@ -73,10 +73,6 @@ double test(std::shared_ptr<boost::property_tree::ptree> params)
       laplace._dof_handler, laplace._constraints, a, material_property);
   mfmg::Hierarchy<DVector> hierarchy(comm, evaluator, params);
 
-  pcout << "Grid complexity    : " << hierarchy.grid_complexity() << std::endl;
-  pcout << "Operator complexity: " << hierarchy.operator_complexity()
-        << std::endl;
```

Comment:
c.f. https://github.com/ORNL-CEES/mfmg/pull/132#discussion_r257847646 that has not been answered yet.

Why did you chose to close #132 and open a new PR?  It makes it hard to track changes and whether review comments have been addressed.

Reply:
I closed #132 because the approach there is different, and I did not want to reuse that PR. I understand the concern about tracking changes.

Reply:
I took an executive decision to remove it as it does not work at all right now, we have no way of calculating of complexities after redefining operators.

Reply:
Kk